app/src/main/java/ch/epfl/qedit/backend/auth/MockAuthService.java

```diff
@@ -59,6 +59,12 @@ public IdlingResource getIdlingResource() {
     private User createCosme() {
         User cosme = new User("Cosme", "Jordan", User.Role.Editor);
         cosme.addQuiz("quiz0", "Qualification EPFL");
+        cosme.addQuiz("quiz1", "Quizz 1");
+        cosme.addQuiz("quiz2", "Quizz 2");
+        cosme.addQuiz("quiz3", "Quizz 3");
+        cosme.addQuiz("quiz4", "Quizz 4");
+        cosme.addQuiz("quiz5", "Quizz 5");
+        cosme.addQuiz("quiz6", "Quizz 6");
```

Comment:
Typo: it's "Quiz"

Reply:
According to inspect code, there is a typo in your family name ;)

Reply:
Oh yeah? That's unacceptable. You know what, I will make a pr with a dictionary :)))

Reply:
Problem resolved !